src/main/java/lotto/game/domain/code/PrizeCode.java

```diff
@@ -0,0 +1,72 @@
+package lotto.game.domain.code;
+
+public enum PrizeCode {
+	NOTHING(6, 0, 0),
+	FIFTH_PLACE(5, 3, 5000),
+	FOURTH_PLACE(4, 4, 50000),
+	THIRD_PLACE(3, 5, 1500000),
+	// SECOND_PLACE(2, 5, 20000000),
+	WINNER(1, 6, 2000000000);
+
+	private int grade;
+	private int countOfMatch;
+	private int winningAmount;
+
+	PrizeCode(int grade, int countOfMatch, int winningAmount) {
+		this.grade = grade;
+		this.countOfMatch = countOfMatch;
+		this.winningAmount = winningAmount;
+	}
+
+	public static PrizeCode findCode(int countOfMatch) {
+		if (WINNER.countOfMatch() == countOfMatch) {
+			return WINNER;
+		}
+		if (THIRD_PLACE.countOfMatch() == countOfMatch) {
+			return THIRD_PLACE;
+		}
+		if (FOURTH_PLACE.countOfMatch() == countOfMatch) {
+			return FOURTH_PLACE;
+		}
+		if (FIFTH_PLACE.countOfMatch() == countOfMatch) {
+			return FIFTH_PLACE;
+		}
+		return NOTHING;
```

Comment:
이 부분은 람다식을 활용해서 표현해 볼 수 있을 것 같습니다 😄 
```suggestion
		return Arrays.stream(values())
			.filter(pc -> pc.countOfMatch == countOfMatch)
			.findFirst()
			.orElse(NOTHING);
```